### PR TITLE
chore(admin): convert useDebounce to TS

### PR DESCRIPTION
**Pull request by gu-stav**
### What does it do?

Convert `useDebounce` to TS.

### CODE REVIEW

File: packages/core/admin/admin/src/hooks/useDebounce.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+export function useDebounce<Value>(value: Value, delay: number): Value {
```
Comment (joshuaellis) [inline packages/core/admin/admin/src/hooks/useDebounce.ts:3]:
```suggestion
export function useDebounce<TValue>(value: TValue, delay: number): TValue {
```

iirc we're appending generics with T to not be confused with types 👍🏼

**Review by joshuaellis — APPROVED** [review]

**Review by joshuaellis — APPROVED** [review]

**Comment by gu-stav** [comment]
@joshuaellis sorry - GH is having issues at the moment and the PRs are stale :/

**Review by Boegie19 — APPROVED** [review]

**Comment by Boegie19** [comment]
@gu-stav can you merge it now since I did an approved?

**Comment by joshuaellis** [comment]
> @gu-stav can you merge it now since I did an approved?

your approvals don't count unfortunately :(

**Review by joshuaellis — APPROVED** [review]

**Comment by gu-stav** [comment]
There are still problems for PRs, but I think there aren't any pending pushes anymore: https://www.githubstatus.com/incidents/9fk4c2w43clz

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**